BUILD.bazel

```diff
@@ -449,6 +471,15 @@ cc_library(
     ]
 )
 
+java_library(
+    name="ray_java_pkg",
+    srcs=glob([
+        "java/*.java",
+        "java/*.properties",
```

Comment:
I tried this, the generated jar file doesn't capture anything. 
I think this should be `java/**/*.java`, right? 
Also, we need to add the external dependencies.

--------------

A few more comments:
1) instead of defining single big java library, we should probably define each module (api, runtime, test, etc) as a library.
2) We should probably follow this [tutorial](https://docs.bazel.build/versions/master/migrate-maven.html) to properly set up the java libraries.
3) It seems that you want to set up java libraries later. If so, we should probably remove this `ray_java_pkg` from this PR, because it doesn't actually do anything.